### DISCUSSION TITLE
Bypassing Locked Mode On Forms to Cheat

### DISCUSSION THREAD

**Discussion by cooltoo37** [3kh0/ext-remover#1405] [Exploits JOIN TITANIUM NETWORK TO SHARE PLEASE]
Exams are coming up and your teachers are using Google Forms locked mode which prevents you from opening up other tabs to cheat I have found a script that you can use tamper monkey to inject into Google Forms website and it allows you to visit other tabs without the teacher knowing this has worked on my computer that sh1mmer been used to remove other restrictions this script also works on windows I don't know about Mac use it at your own risk keep in mind that using your own Google account on the forms doesn't work so using your school account sign in this is not made by me there are 3 completely different ways to do this one is easy one is hard and complicated 
 [The easy one](https://github.com/xNasuni/google-forms-unlocker?tab=readme-ov-file#using-a-user-script-manager)
 [The medium one](https://github.com/s-pscripts/locked-mode-x)
 [The harder one](https://tinyurl.com/LockedModeBypass2023c)

**Comment by S-PScripts** [comment]
a few months ago i actually found another!
https://github.com/s-pscripts/locked-mode-x/

v4pl also found one here:
https://github.com/S-PScripts/chromebook-utilities/blob/main/Exploits/Locked%20Mode%20Bypass%20-%20V4PL

**Reply by S-PScripts** [reply]
also https://docs.google.com/spreadsheets/d/14gzy3LHMLjvRpEl7SuGcMvfIlm2hZ5jRCY_MeBY0haE/edit?gid=0#gid=0 doesn't work @cooltoo37

**Reply by cooltoo37** [reply]
this link is the one I have mentioned and called it the easier one but might not work for everyone https://github.com/s-pscripts/locked-mode-x/ 
the second one which doesn't work is v4pl sheet which you can access by using this url that works https://tinyurl.com/LockedModeBypass2023c I just had the forms link so it did'int work its the same thing let me know if you found any other ways I had 2 exams today in which the easy method the first one worked flawlessly with no issue trying to find a better one and one that works with restricted computers keep me updated.

**Reply by S-PScripts** [reply]
> this link is the one I have mentioned and called it the easier one but might not work for everyone https://github.com/s-pscripts/locked-mode-x/ the second one which doesn't work is v4pl sheet which you can access by using this url that works https://tinyurl.com/LockedModeBypass2023c I just had the forms link so it did'int work its the same thing let me know if you found any other ways I had 2 exams today in which the easy method the first one worked flawlessly with no issue trying to find a better one and one that works with restricted computers keep me updated.

in your original post "the easier one" wasn't mine though
but I assume "the harder one" was your copy of v4pl's

**Reply by S-PScripts** [reply]
@cooltoo37 i edited your original post

**Reply by cooltoo37** [reply]
> > this link is the one I have mentioned and called it the easier one but might not work for everyone https://github.com/s-pscripts/locked-mode-x/ the second one which doesn't work is v4pl sheet which you can access by using this url that works https://tinyurl.com/LockedModeBypass2023c I just had the forms link so it did'int work its the same thing let me know if you found any other ways I had 2 exams today in which the easy method the first one worked flawlessly with no issue trying to find a better one and one that works with restricted computers keep me updated.
> 
> in your original post "the easier one" wasn't mine though but I assume "the harder one" was your copy of v4pl's

yes exactly harder version is v4pl's

**Comment by Charlieis1nsan3** [comment]
bruh did you just watch my video?

**Reply by S-PScripts** [reply]
@Charlieis1nsan3 you forgot to blur out your email address in your locked mode video

**Reply by Charlieis1nsan3** [reply]
I don't really care. its not like anybody can do anything with it, most of the people on this repository is a skid. -_-

**Reply by Charlieis1nsan3** [reply]
(most)

**Reply by cooltoo37** [reply]
no I haven't watched any video I came across these on Google

**Comment by OPcoolman** [comment]
Lol Just do your own work lmao

**Comment by xNasuni** [comment]
you can use my script [xNasuni/google-forms-unlocker](https://github.com/xnasuni/google-forms-unlocker/) if you wanna do your quizzes/tests without being in locked mode !! :3

**Comment by SlitheringLizard1** [comment]
most managed chromebooks do not have inspect.
